src/amo/components/EditableCollectionAddon/index.js

```diff
@@ -56,11 +111,59 @@ export class EditableCollectionAddonBase extends React.Component<Props> {
             {i18n.gettext('Remove')}
           </Button>
         </div>
+        {showNotes && (
+          <div className="EditableCollectionAddon-notes">
+            <h4 className="EditableCollectionAddon-notes-header">
+              <Icon name="comments-blue" />
+              {i18n.gettext('User comment')}
+            </h4>
+
+            {this.state.editingNote ? (
+              <React.Fragment>
+                {errorHandler.renderErrorIfPresent()}
+                <DismissibleTextForm
+                  className="EditableCollectionAddon-notes-form"
+                  microButtons
+                  onDelete={this.onDeleteNote}
+                  onDismiss={this.onDismissNoteForm}
+                  onSubmit={this.onSaveNote}
+                  placeholder={i18n.gettext('Add a comment about this add-on.')}
+                  submitButtonText={i18n.gettext('Save')}
+                  text={addon.notes}
+                />
+              </React.Fragment>
+            ) : (
+              <div className="EditableCollectionAddon-notes-read-only">
+                <span
+                  className="EditableCollectionAddon-notes-content"
+                  // eslint-disable-next-line react/no-danger
+                  dangerouslySetInnerHTML={sanitizeHTML(addon.notes)}
```

Comment:
should we respect new lines? If so, we should probably use `nl2br` and white-list `<br>`.